### PR TITLE
Update students.yml

### PR DESCRIPTION
**Pull request by ariqzuhdi**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# *Do not* delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -748,3 +748,13 @@
   image: danipop.jpg
   bio: Christian, supporter of traditional family(Man + Woman), coder, sportsman. In this order
   facebook: dani.avram.pop
+  
+- name: Ariq Zuhdi
+  github: ariqzuhdi
+  image: ariqzuhdi.jpg
```
Comment (robbyoconnor) [inline _data/students.yml:754]: Where is this image @ariqzuhdi ?

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

**Comment by ariqzuhdi** [comment]
In the images/students sir.

**Comment by robbyoconnor** [comment]
Is it? I don't see it in the PR.

**Comment by robbyoconnor** [comment]
Look at the included files.

**Comment by robbyoconnor** [comment]
Please update this pull request with all the information. You never uploaded the file.

**Comment by robbyoconnor** [comment]
@ariqzuhdi -- Also you didn't even bother to read the pull request template. Go read that.

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please and your image in 240x240 pixels and read the PR body (template) pls

**Comment by robbyoconnor** [comment]
@Abhi2424shek what image?

**Comment by ariqzuhdi** [comment]
Sorry i was sleepy, i. Fix it as soon as possible

On Jan 5, 2017 9:08 AM, "robbyoconnor" <notifications@github.com> wrote:

@Abhi2424shek <https://github.com/Abhi2424shek> what image?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/fossasia/gci16.fossasia.org/pull/826#issuecomment-270545006>,
or mute the thread

**Comment by robbyoconnor** [comment]
No problem -- you need sleep -- you're young!

**Comment by ariqzuhdi** [comment]
Umm, i have added it sir.

**Comment by robbyoconnor** [comment]
@ariqzuhdi No you did not. Add it to the `patch-1` branch.

**Comment by ariqzuhdi** [comment]
Ahh sorry, wrong branch >.<

**Comment by robbyoconnor** [comment]
Build failure isn't your fault.

**Pull request merged**